playwright/data/e2eTestData.ts

```diff
@@ -80,18 +46,45 @@ export const COUNTRIES = {
     name: "Bosnia and Herzegovina",
   },
 };
-export const CHANNELS = {
-  channelToBeEditedSettings: {
-    id: "Q2hhbm5lbDoyMzkx",
+export const CUSTOMERS = {
+  deleteCustomer: {
+    // id: "VXNlcjoxNzM=",
+    id: "VXNlcjo5MTA=",
```

Comment:
Looks like a leftover?
```suggestion
  deleteCustomer: {
    id: "VXNlcjo5MTA=",
```